docs/source/index.mdx

```diff
@@ -15,10 +15,10 @@ limitations under the License.
 -->
 
 
-# Optimum Habana
+# 🤗 Optimum Habana
 
 🤗 Optimum Habana is the interface between the 🤗 Transformers library and [Habana's Gaudi processor (HPU)](https://docs.habana.ai/en/latest/index.html). It provides a set of tools enabling easy model loading and fine-tuning on single- and multi-HPU settings for different downstream tasks.
-The current release focuses on question answering and text classification and enables users to try other models for other tasks with only a few changes.
+The list of officially supported models and tasks is available [here](validated-models). Users can try other models and tasks with only few changes.
```

Comment:
same here, should we change supported to sth else?

Reply:
Same reply with "validated", wdyt?

Reply:
yes